### PR TITLE
Issue #1303 - added section for poor performance after migration

### PR DESCRIPTION
**Pull request by bcrisologo**
#1303 issue - added a section where poor performance is experienced after migration on pg service.  Modified the formatting and some language from original article.

### CODE REVIEW

File: docs/products/postgresql/howto/migrate-pg-dump-restore.rst [hunk 1]
```diff
@@ -92,3 +92,13 @@ A typical example is due to failing ``COMMENT ON EXTENSION`` queries trying to r
...
     ``pg_restore`` offers similar ``--no-XXX`` options to switch off other, often unnecessary restore queries. More information is available in the `PostgreSQL documentation <https://www.postgresql.org/docs/current/app-pgrestore.html>`_.
+
+
+Poor performance after migration
+--------------------------------
+Whenever you load data with the ``pg_restore`` or similar tools, it is recommended to run ``ANALYZE`` or ``VACUUM ANALYZE`` on your entire database to collect new statistics.  The database will not have up-to-date statistics on the tables and indexes without these operations.  In turn, this may lead to poor query plans and poor database performance.
+
+Generally, the Aiven platform automatically runs ``ANALYZE`` on your service after performing a major version upgrade in order to ensure the statistics are up-to-date.
+
+For more information about ``ANALYZE``, you may see the official `sql analyze <https://www.postgresql.org/docs/current/sql-analyze.html>`_ documentation.
```
Comment (ftisiot) [inline docs/products/postgresql/howto/migrate-pg-dump-restore.rst:103]: I believe SQL should be capital :D

**Review by ftisiot — CHANGES_REQUESTED** [review]
Hi @bcrisologo,
Thanks for this, one small fix before it's ready to be merged

File: docs/products/postgresql/howto/migrate-pg-dump-restore.rst (hunk 1 above)
Comment (bcrisologo) [reply docs/products/postgresql/howto/migrate-pg-dump-restore.rst]: Updated :)

**Review by ftisiot — APPROVED** [review]
LGTM

**Pull request merged**